app/src/main/java/com/android/feedme/ui/camera/Camera.kt

```diff
@@ -253,7 +252,7 @@ fun takePhoto(
 }
 
 /** Check if the app has the required permissions to use the camera. */
-fun hasRequiredPermissions(context: Context): Boolean {
+private fun hasRequiredPermissions(context: Context): Boolean {
```

Comment:
Thanks for helping me notice this line, I think I'm going to move the `hasRequiredPermissions` method to the `CameraViewModel` in a future PR.

Reply:
Nice catch !

Reply:
For the Gallery Screen, I will make a more appropriate implementation along with the display of selected pictures